recipes/ffmpeg/all/conanfile.py

```diff
@@ -0,0 +1,348 @@
+    homepage = "https://ffmpeg.org"
+    topics = ("ffmpeg", "multimedia", "audio", "video", "encoder", "decoder", "encoding", "decoding",
+             "transcoding", "multiplexer", "demultiplexer", "streaming")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "postproc": [True, False],
+               "zlib": [True, False],
+               "bzlib": [True, False],
+               "lzma": [True, False],
+               "iconv": [True, False],
+               "freetype": [True, False],
+               "openjpeg": [True, False],
+               "openh264": [True, False],
+               "opus": [True, False],
+               "vorbis": [True, False],
+               "zmq": [True, False],
+               "sdl2": [True, False],
+               "x264": [True, False],
+               "x265": [True, False],
+               "vpx": [True, False],
+               "mp3lame": [True, False],
+               "fdk_aac": [True, False],
+               "webp": [True, False],
+               "openssl": [True, False],
+               "alsa": [True, False],
+               "pulse": [True, False],
+               "vaapi": [True, False],
+               "vdpau": [True, False],
+               "xcb": [True, False],
+               "appkit": [True, False],
+               "avfoundation": [True, False],
+               "coreimage": [True, False],
+               "audiotoolbox": [True, False],
+               "videotoolbox": [True, False],
+               "securetransport": [True, False],
+               "qsv": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "postproc": True,
+                       "zlib": True,
+                       "bzlib": True,
+                       "lzma": True,
+                       "iconv": True,
+                       "freetype": True,
+                       "openjpeg": True,
+                       "openh264": True,
+                       "opus": True,
+                       "vorbis": True,
+                       "zmq": False,
+                       "sdl2": False,
+                       "x264": True,
+                       "x265": True,
+                       "vpx": True,
+                       "mp3lame": True,
+                       "fdk_aac": True,
+                       "webp": True,
+                       "openssl": True,
+                       "alsa": True,
+                       "pulse": True,
+                       "vaapi": True,
+                       "vdpau": True,
+                       "xcb": True,
+                       "appkit": True,
+                       "avfoundation": True,
+                       "coreimage": True,
+                       "audiotoolbox": True,
+                       "videotoolbox": True,
+                       "securetransport": False,  # conflicts with OpenSSL
```

Comment:
Would it make sense to merge openssl & securetransport in only one option?

Reply:
You mean like an option `ssl` with various option? None, openssl or securetransport?

Reply:
False, openssl, securetransport